Makefile

```diff
@@ -34,6 +34,24 @@ env: kafl/env.sh
 	@echo "Entering environment in sub-shell. Exit with 'Ctrl-d'."
 	@PROMPT_COMMAND='source kafl/env.sh; unset PROMPT_COMMAND' $(SHELL)
 
+define HELP_TEXT
```

Comment:
Good help text !
`Supported actions` is hard to differentiate with the `Developer actions`,
maybe rename that as simply `actions` ? or `User actions` ?

Reply:
done